enterprise/user-code-namespaces/user-code-namespaces-client/src/main/java/com/hazelcast/namespaces/dyamicconfig/Client.java

```diff
@@ -0,0 +1,26 @@
+package com.hazelcast.namespaces.dyamicconfig;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.config.UserCodeNamespaceConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.map.EntryProcessor;
+import usercodenamespaces.IncrementingEntryProcessor;
+
+public class Client {
+
+    public static void main(String[] args) {
+        HazelcastInstance client = HazelcastClient.newHazelcastClient();
```

Comment:
Do these need to be shut down afterwards?

Reply:
Yes